src/js/views/hallOfFame.js

```diff
@@ -30,11 +30,19 @@ async function updatePlayers(inputs, updateEvents) {
                 min: 0,
                 per: 0,
             };
+            players[i].teamSums = {};
             for (let j = 0; j < players[i].stats.length; j++) {
+                const team = players[i].stats[j].tid;
```

Comment:
Call this variable `tid`, `team` is generally used to refer to the core/team module.